src/components/tables/TableDataModifier.ts

```diff
@@ -103,16 +103,27 @@ export const applySortOnDisplayedIds = (
   tableCompositeState: ITableCompositeState,
```

Comment:
If there's a quick win to have less function params I would do it since it looks quite tricky to call the function. Not a major issue, just refactor proposition to keep in mind.

Reply:
yeah this has been made in the context of a very big PR, there is surely room for improvement in these methods